Forge Standard Library docs (#207)
foundry-rs/book: src/reference/forge-std/zeroVarError.md

## `zeroVarError`

### Signature

```solidity
stdError.zeroVarError
```

### Description

> The error when trying to call a function via an in-function function pointer that has not been initialized.

Reviewer: ```suggestion
The internal Solidity error when trying to call a function via a function pointer that has not been initialized.
```
